sympy/geometry/tests/test_line.py

```diff
@@ -699,3 +699,14 @@ def _check():
     # midline intersection
     c, d = (-2, -3), (-2, 0)
     _check()
+
+
+def tes_issue_13705
+x = Symbol('x')
```

Comment:
* the file needs to end with a newline
* the lines of the function need to be indented
* see this file (or others) for an example where `raises(ValueError,...)` is used..